charts/internal/shoot-system-components/charts/rook-ceph/templates/ceph-cluster.yaml

```diff
@@ -0,0 +1,249 @@
+#################################################################################################################
+ # Define the settings for the rook-ceph cluster with common settings for a production cluster.
+ # All nodes with available raw devices will be used for the Ceph cluster. At least three nodes are required
+ # in this example. See the documentation for more details on storage settings available.
+
+ # For example, to create the cluster:
+ #   kubectl create -f crds.yaml -f common.yaml -f operator.yaml
+ #   kubectl create -f cluster.yaml
+ #################################################################################################################
+
+ apiVersion: ceph.rook.io/v1
+ kind: CephCluster
+ metadata:
+   name: rook-ceph
+   namespace: rook-ceph # namespace:cluster
+ spec:
+   cephVersion:
+     # The container image used to launch the Ceph daemon pods (mon, mgr, osd, mds, rgw).
+     # v13 is mimic, v14 is nautilus, and v15 is octopus.
+     # RECOMMENDATION: In production, use a specific version tag instead of the general v14 flag, which pulls the latest release and could result in different
+     # versions running within the cluster. See tags available at https://hub.docker.com/r/ceph/ceph/tags/.
+     # If you want to be more precise, you can always use a timestamp tag such ceph/ceph:v15.2.8-20201217
+     # This tag might not contain a new Ceph version, just security fixes from the underlying operating system, which will reduce vulnerabilities
+     image: ceph/ceph:v15.2.8
```

Comment:
This should not be hard-coded here but a chart value and populated from the image vector

Reply:
I think I got it right, take a look.

Reply:
chiming in here: It's very much recommended to use the exact date-time tags for ceph as the others might be rebuild. i.e. instead of `15.2.8` use `ceph/ceph:v15.2.13-20210526`

Reply:
Sure.

Reply:
Done